src/test/kotlin/biokotlin/cli/TestExtension.kt

```diff
@@ -0,0 +1,39 @@
+package biokotlin.cli
+
+import biokotlin.util.setupDebugLogging
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import java.io.File
+
+class TestExtension : BeforeAllCallback {
```

Comment:
I would recommend not using a TestExtension here.  I think the purpose of the TestExtension is when you have multiple unit tests reading and writing the same files/folders.  It makes more sense with the PHG as we tend to have Integration Tests walking over the same files we use in the unit tests.  If we use the test extension for all single use files for the unit tests it will get very large.

Reply:
Ok, I will move the constents